controllers/user.controller.js

```diff
@@ -22,101 +21,93 @@ exports.create = (req, res) => {
     email: req.body.email,
     password: hash,
   };
-
-  User.findAll({where: {email: user.email}}).then(exist => {
-    if(exist.length > 0){
-        console.log(exist)
-        return res.status(422).send({
-            message: 'Email has already been taken.'
-        })
-    }else{
-        User.create(user)
-        .then((data) => {
-          res.send(data);
-        })
-        .catch((err) => {
-          res.status(500).send({
-            message: err.message || "Some error occurred while creating new user.",
-          });
-        });
+  try {
+    const userUniqueEmailCheck = await User.findAll({
+      where: { email: user.email },
+    });
+    if (userUniqueEmailCheck.length > 0) {
+      return res.status(422).send({
+        message: "Email has already been taken.",
+      });
+    } else {
+      const newUser = await User.create(user);
+      return res.send(newUser);
     }
-  })
-
+  } catch (err) {
+    return res.status(500).send({
+      message: err.message || "Some error occurred while creating new user.",
+    });
+  }
 };
 
-exports.findAll = (req, res) => {
-  User.findAll()
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      res.status(500).send({
-        message:
-          err.message || "Some error occurred while while looking for users.",
-      });
+exports.findAll = async (req, res) => {
+  try {
+    const users = await User.findAll();
```

Comment:
res.send(await User.findAll())

Reply:
imo to juz kwestia dyskusyjna, tak jak zrobil Kacper jest moim zdaniem czytelniej